### DISCUSSION TITLE
LLC resource hungry?

### DISCUSSION THREAD

**Discussion by YouveGotMeowxy** [JulianPrieber/llc-docker#15] [Issues]
Using the latest container, while even just idling, LLC uses a disproportionate amount of memory and CPU (compare it with the other running apps' usages, and this is technically just a relatively "dead listing" of links)?

![image](https://user-images.githubusercontent.com/39491297/187057774-edc9b774-af5f-42dd-b722-59ed673488f2.png)

For example, it uses as much as Nextcloud and Sonarr's memory *combined*

Note: screenshot was taken while LLC had been idling for hours.

**Comment by JulianPrieber** [comment]
If your CPU is running at this level, consistently, something is definitely wrong.
LLC app itself does not run any background tasks, there should only be a couple of spikes now and then.

You can run the command `top` from your docker console to get a bit more information.

<br>

Memory consumption is a bit high, but not alarming. As long as the memory usage is not going up consistently and not down again, this is probably not the end of the world.

The docker mostly uses the default Apache configuration, which isn't really memory efficient. You can certainly optimize that with some tinkering in the Apache config.

Alternatively, [Docker supports setting a memory limit for individual docker containers](https://docs.docker.com/config/containers/resource_constraints/#limit-a-containers-access-to-memory). The docker should run fine about 220 MB of RAM.
